src/libs/shared_headers/include/shared/events.h

```diff
@@ -19,4 +19,12 @@
 // crosshair
 #define TELESCOPE_ACTIVE "TelescopeActive"
 
+// Worldmap
+#define WORLDMAP_ENCOUNTER_TYPE_UNKNOWN 0
+#define WORLDMAP_ENCOUNTER_TYPE_MERCHANT 1
+#define WORLDMAP_ENCOUNTER_TYPE_WARRING 2
+#define WORLDMAP_ENCOUNTER_TYPE_FOLLOW 3
+#define WORLDMAP_ENCOUNTER_TYPE_STORM 4
+#define WORLDMAP_ENCOUNTER_TYPE_INVALID 2147483647
```

Comment:
do the scripts support hex literals? if so, `0x7ffffff` would be much more self-explanatory...

Reply:
I've also added a small comment for anyone who might be print-debugging this

Reply:
Turns out scripts don't support hex numbers, and it's very important to re-copy these headers when testing this.